### PR TITLE
feat: getAccessKey method

### PR DESCRIPTION
**Pull request by murka**

### CODE REVIEW

File: src/server_manager/web_app/testing/models.ts [hunk 1]
```diff
@@ -127,6 +127,9 @@ export class FakeServer implements server.Server {
   getVersion() {
     return '1.2.3';
   }
+  getAccessKey(_accessKeyId: server.AccessKeyId) {
```
Comment (fortuna) [inline src/server_manager/web_app/testing/models.ts:130]: This is not being used in the Manager. Please revert the server manager changes.

File: src/shadowbox/model/access_key.ts [hunk 2]
```diff
@@ -53,6 +53,8 @@ export interface AccessKeyRepository {
   createNewAccessKey(): Promise<AccessKey>;
   // Removes the access key given its id. Throws on failure.
   removeAccessKey(id: AccessKeyId);
+  // Returns the access key with the given id. Throws on failure.
+  getAccessKey(id: AccessKeyId): AccessKey;
```
Comment (fortuna) [inline src/shadowbox/model/access_key.ts:57]: This is a reasonable addition to the API, and your code changes look solid. Thanks for carefully putting this together.

Could you clarify why you need this addition? Usually the listAccessKeys() is enough, so I'd like to better understand what the underlying need is.

Comment (fortuna) [inline src/shadowbox/model/access_key.ts:57]: We need to update https://github.com/Jigsaw-Code/outline-server/blob/master/src/shadowbox/server/api.yml too

File: src/shadowbox/server/manager_service.ts [hunk 3]
```diff
@@ -127,6 +127,8 @@ export function bindService(
     service.setPortForNewAccessKeys.bind(service)
   );
 
+  apiServer.get(`${apiPrefix}/access-keys/:id`, service.getAccessKey.bind(service));
+
   apiServer.post(`${apiPrefix}/access-keys`, service.createNewAccessKey.bind(service));
   apiServer.get(`${apiPrefix}/access-keys`, service.listAccessKeys.bind(service));
```
Comment (fortuna) [inline src/shadowbox/server/manager_service.ts:134]: Let's move the get key to after the list keys, so all the ` access-keys/:id` entries are together.

```suggestion
  apiServer.post(`${apiPrefix}/access-keys`, service.createNewAccessKey.bind(service));
  apiServer.get(`${apiPrefix}/access-keys`, service.listAccessKeys.bind(service));

  apiServer.get(`${apiPrefix}/access-keys/:id`, service.getAccessKey.bind(service));
```

File: src/shadowbox/model/access_key.ts (hunk 2 above)
Comment (murka) [reply src/shadowbox/model/access_key.ts:57]: @fortuna 

Thanks for feedback.
I think this addition need for developing a 3rd-party API, for example I made a [npm package](https://github.com/murka/outlinevpn-api) to interact with outline server, but I must be use array filtering for look up user.

File: src/server_manager/web_app/testing/models.ts (hunk 1 above)
Comment (murka) [reply src/server_manager/web_app/testing/models.ts]: Done.

File: src/shadowbox/model/access_key.ts (hunk 2 above)
Comment (murka) [reply src/shadowbox/model/access_key.ts:57]: @fortuna

FYI https://github.com/Jigsaw-Code/outline-server/pull/1142/commits/234155e9ae6438c39e25cc42f874a3dd0af800f0

**Comment by murka** [comment]
@googlebot I signed it!

File: src/server_manager/web_app/testing/models.ts (hunk 1 above)
Comment (fortuna) [reply src/server_manager/web_app/testing/models.ts]: I still see the Manager changes there: 
image

Please revert those.

**Review by fortuna — CHANGES_REQUESTED** [review]

**Comment by murka** [comment]
@fortuna @daniellacosse up

**Comment by fortuna** [comment]
It seems the tests are still failing for this PR
image

**Review by fortuna — APPROVED** [review]

**Comment by fortuna** [comment]
Merged. Thank you for your contribution!

**Pull request merged**